whois-api/src/test/java/net/ripe/db/whois/api/fulltextsearch/ElasticFullTextSearchTestIntegration.java

```diff
@@ -249,11 +248,12 @@ public void search_multiple_results_with_search_limit() {
 
          rebuildIndex();
 
-        final QueryResponse queryResponse = query("q=remark&facet=true");
+        final QueryResponse queryResponse = query("q=remark&facet=true&rows=3");
 
-        //search limit to 3
+        //search limit to 3, however the total that ES is able to find is 5
         assertThat(queryResponse.getStatus(), is(0));
-        assertThat(queryResponse.getResults().getNumFound(), is(3L));
```

Comment:
If you set the system property limit as 10 instead of 3 this will anyways return 5 objects? I think the behaviour is changed now it checking the row number and object instead of limit. Can you rename the method?

Reply:
- If you set the system property limit as 10 instead of 3 this will anyways return all the objects? Yes, It would return all of them (5)
- I think the behaviour is changed now it checking the row number and object instead of limit. -> Yes, we are not checking the limit I will change the name

Reply:
Well, we are limiting the output right? Not in this test case limit never came into picture here as there are just  5 objects and limit is 10 so it will never breach it

Reply:
I renamed by search_multiple_results_with_first_three_records()